app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,16 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:email])
+    # require "pry"; binding.pry
+    if user != nil && user.authenticate(params[:password])
```

Comment:
Add the creation of a session when a merchant user logs in, or anyone logs in for that matter